ui/templates/search/indexes/learningresources/learningresource_text.txt

```diff
@@ -0,0 +1,3 @@
+{{ object.title }}
```

Comment:
Sorry, it was my misunderstanding.  I thought it was the template to show the search result.